keepalived/check/check_parser.c

```diff
@@ -115,12 +115,42 @@ lbalgo_handler(vector_t *strvec)
 	char *str = vector_slot(strvec, 1);
 	int size = sizeof (vs->sched);
 	int str_len = strlen(str);
-
 	if (size > str_len)
 		size = str_len;
-
 	memcpy(vs->sched, str, size);
 }
+
+static void
+lbflags_handler(vector_t *strvec)
+{
+	virtual_server_t *vs = LIST_TAIL_DATA(check_data->vs);
+	int i;
+	char* str;
+	vector_foreach_slot_from(strvec,str,i,1)
+	{
+		if (!strcmp(str, "persistant"))
+			vs->flags |= IP_VS_SVC_F_PERSISTENT;
+		if (!strcmp(str, "hashed"))
```

Comment:
Persistence can already be set using keywords `persistence_timeout` and/or `persistence_granularity`